app/javascript/modules/listingDetails/ListingDetailsEligibility.tsx

```diff
@@ -199,10 +199,12 @@ export const ListingDetailsEligibility = ({
                       : `${unit.numberOfUnits} ${defaultIfNotTranslated("t.units", "units")}`
                   }
                 >
-                  {defaultIfNotTranslated(
-                    `listings.unitsHaveAccessibilityFeaturesFor.${unit.name}`,
-                    `These units have accessibility features for people with ${unit.name}.`
-                  )}
+                  <p className="text-tiny">
```

Comment:
According to the comments in Jira, looks like this just needs `text-gray-700`, then it's good to go